lib/Browser.js

```diff
@@ -45,8 +89,24 @@ class Browser extends EventEmitter {
    */
   async newPage() {
     const {targetId} = await this._connection.send('Target.createTarget', {url: 'about:blank'});
-    const client = await this._connection.createSession(targetId);
-    return await Page.create(client, this._ignoreHTTPSErrors, this._appMode, this._screenshotTaskQueue);
+    const {targetInfo} = await this._connection.send('Target.getTargetInfo', {targetId});
+    const page = await this._targetForInfo(targetInfo).page();
+    await page.goto('about:blank');
+    return page;
+  }
+
+  /**
+   * @return {Array<Target>}
```

Comment:
For the sake of consistency, please either:
- use `'!'` for non-nullable types in this patch
- remove all exclamation marks throughout the code first